src/skyparams.h

```diff
@@ -40,6 +40,7 @@ struct SkyboxParams
 	video::SColor fog_sun_tint;
 	video::SColor fog_moon_tint;
 	std::string fog_tint_type;
+	float body_orbit_tilt;
```

Comment:
This is potentially uninitialized - not mentioned in SkyboxDefaults::getSkyDefaults()

Reply:
It's at line # 99 in the same file (one chunk further)

Reply:
Pushed an amend to eradicate this setting entirely.